pallets/nfts/src/common_functions.rs

```diff
@@ -92,6 +92,55 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		Self::deposit_event(Event::NextCollectionIdIncremented { next_id });
 	}
 
+	/// Increment the number of items in the `collection` owned by the `owner`. If no entry exists
+	/// for the `owner` in `AccountBalance`, create a new record and reserve `deposit_amount` from
+	/// the `deposit_account`.
+	pub(crate) fn increment_account_balance(
+		collection: T::CollectionId,
+		owner: &T::AccountId,
+		(deposit_account, deposit_amount): (
```

Comment:
Note: Not possible to reuse the tuple type `AccountDepositOf` with `AccountId` passed by reference.